test-series/1/test-1.0.0.ipynb

```diff
@@ -0,0 +1,1449 @@
+{
```

Comment:
<p>Isn't this already provided by <code>astropy.cosmology.Planck15</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HERA-Team/hera-validation/pull/51/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>It's very very slightly different, and this is exactly what went into the original.</p><p> </p>